assets/chezmoi.io/docs/user-guide/frequently-asked-questions/encryption.md

```diff
@@ -100,3 +100,54 @@ $ chezmoi add --encrypt ~/.ssh/id_rsa
 When you run `chezmoi init` on a new machine you will be prompted to enter your
 passphrase once to decrypt `key.txt.age`. Your decrypted private key will be
 stored in `~/.config/chezmoi/key.txt`.
+
+## How to migrate between encryption tools?
+
+To migrate from gpg to age encryption or vise versa, one can follow these steps:
+
+1. Make sure you have all encrypted files applied (e.g. `chezmoi apply` decrypts files and places them in their destinations).
+2. Update chezmoi configuration to use the new encryption method
+   (examples: [gpg](/user-guide/encryption/gpg), [age](/user-guide/encryption/age), [age with one-time passphrase](#how-do-i-configure-chezmoi-to-encrypt-files-but-only-request-a-passphrase-the-first-time-chezmoi-init-is-run)).
+3. Remove all encrypted files from the state via `chezmoi forget` or `chezmoi unmanage`.
+4. Add them back with `chezmoi add --encrypt`.
+
+### Example: migrate from gpg to age
```

Comment:
Prefer title case: `Example: Migrate from GPG to age`; you could make it `` Example: Migrate from `gpg` to `age` ``.

Reply:
I agree.